README.md

```diff
@@ -5,6 +5,13 @@
 
 Croodle is an end-to-end encrypted web application to schedule a date or to do a poll on a any topic. All data like title, description, number and labels of options, available answers and names of users and their selections are encrypted/decrypted in the browser using strong 256-bit AES encryption.
 
+[<img alt="Croodle Type" src="screenshots/0_type.png" width="30%">](screenshots/0_type.png)
+[<img alt="Croodle Title and Description" src="screenshots/1_title_desc.png" width="30%">](screenshots/1_title_desc.png)
+[<img alt="Croodle Days" src="screenshots/2_days.png" width="30%">](screenshots/2_days.png)
+[<img alt="Croodle Times" src="screenshots/3_times.png" width="30%">](screenshots/3_times.png)
+[<img alt="Croodle Settings" src="screenshots/4_settings.png" width="30%">](screenshots/4_settings.png)
+[<img alt="Croodle Settings" src="screenshots/5_votes.png" width="30%">](screenshots/5_votes.png)
```

Comment:
Can you please add meaningful description of the images in the `alt` attribute?

Reply:
@jelhan how about now?